packages/beacon-node/test/spec/general/bls.ts

```diff
@@ -139,9 +152,16 @@ function eth_fast_aggregate_verify(input: {pubkeys: string[]; message: string; s
 function fast_aggregate_verify(input: {pubkeys: string[]; message: string; signature: string}): boolean | null {
   const {pubkeys, message, signature} = input;
   try {
-    return bls.Signature.fromBytes(fromHexString(signature), undefined, true).verifyAggregate(
-      pubkeys.map((hex) => bls.PublicKey.fromBytes(fromHexString(hex), CoordType.jacobian, true)),
-      fromHexString(message)
+    const sig = Signature.deserialize(fromHexString(signature), undefined);
+    sig.sigValidate();
```

Comment:
need to search for all ` sigValidate()` and replace with `signatureFromBytes()`

Reply:
Ok. Good idea!

Reply:
Updated.